collective/pwexpiry/pwexpiry_plugin.py

```diff
@@ -76,7 +76,7 @@ def authenticateCredentials(self, credentials):
             return None
 
         password_date = user.getProperty('password_date', '2000/01/01')
-        if str(password_date) != '2000/01/01':
+        if password_date != DateTime('2000/01/01'):
```

Comment:
as we are storing passwort_date with type date in `memberdata_properties.xml`

```xml
<property name="password_date" type="date">2000/01/01</property>
```

this looks more sane to me